### ISSUE TITLE
[Issue]: GH2 Whammy not working

### ISSUE THREAD

**Issue by patricksouza472** [Santroller/Santroller#202]
### Please describe your issue.

I have configured my pico as a Santroller USB Adapter, I'm using my Rock Band guitar as a GH guitar and it works fine with GH3. GH2 partially works, the only thing missing is the Whammy bar support. RGH Xbox 360 with usbdsecpatch as nand patch. 

### Version

Latest

### What operating system are you using?

Windows 11

### What microcontroller are you using?

pi pico

### How is your controller programmed?

other issue not controller related or I have not gotten this far

**Comment by sanjay900** [comment]
Did you calibrate the whammy within santroller

**Comment by patricksouza472** [comment]
> Did you calibrate the whammy within santroller

Yes I did. Works perfectly on any other GH game (tested on GH3, GHWT and GH WoR)

**Comment by sanjay900** [comment]
What about calibrating whammy in game (or is that only a thing on PS2)

**Comment by patricksouza472** [comment]
> What about calibrating whammy in game (or is that only a thing on PS2)

Didn't try that, I didn't even know it was a thing on PS2 since I only played with the controllers back then. I'll try it and report back

**Comment by patricksouza472** [comment]
> > What about calibrating whammy in game (or is that only a thing on PS2)
> 
> Didn't try that, I didn't even know it was a thing on PS2 since I only played with the controllers back then. I'll try it and report back

No dice, there's no option for it

**Comment by sanjay900** [comment]
Have you got the guitar in rock band mode, or guitar hero mode?

**Comment by sanjay900** [comment]
Do you mind trying this again with later versions, i wonder if this has changed at all

**Comment by msuts** [comment]
I am on the latest version of Santroller and seeing the same issue with no GH2 whammy.  Whammy works in GH3 and presumably the other games as well. Haven't yet tested in RB1/2/3. The device is a Sparkfun Pro Micro 5V. All other functions work in GH2, just not whammy. Usbsecdpatch is running through DashLaunch.

**Comment by sanjay900** [comment]
what does your calibration look like? seems to me that GH2 PS2 is just very very specific with what whammy values it likes, ive had reports that it works for some and doesn't for others

**Comment by sanjay900** [comment]
Fixed with v10.1.315
